src/omero/plugins/duplicate.py

```diff
@@ -61,6 +72,38 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Duplicate
 
+    def _pre_objects(self, parser):
+        parser.add_argument(
+            "--duplicate-classes",
```

Comment:
I could see dropping `--classes` here and perhaps adding a short-cut, `-D`.

Reply:
It's the largely default anyway.

Reply:
I don't assume there's a way to list what the server will do by default, is there?

Reply:
This is basically what `--dry-run` does but, generally, the name of the subcommand is a big hint.

Reply:
Hmm... I don't know. ```omero duplicate --dry-run Image:160 --report
omero.cmd.Duplicate Image:160 Dry run performed
...
Duplicates
  Arc:107
  Detector:22
  DetectorSettings:22``` doesn't really give me a sense of the default parameters to know what I can change from "duplicate" to "reference". _Are_ there any that default to "reference" (or even "ignore")?

Reply:
Nothing's jumping to my mind apart from things like how `details.owner` are effectively ignored but that can't be overridden with `--duplicate` anyway. Generally the only "do a different thing by default" going on with graphs operations is when it switches to deleting, e.g., removing somebody else's comments when you move an image into a private group (also not overridable), but duplicate never switches to deletion.